### ISSUE TITLE
This your first issue

### ISSUE THREAD

**Issue by chaws** [gabalves/learning_php_udemy#1]
Here's what you'll have to accomplish using PHP only:

Create one php file for each of the following tasks:
* Make a script that prints a "hello world" message (without apache or nginx!)
* Make a script that create a file name "lala" with the contents "this is my lala file"
* Make a script that creates a zip the "lala" file created in the task above and name it "lala.zip"

**Comment by chaws** [comment]
@gabalves updates?

**Comment by gabalves** [comment]
Yes, but i forgot to go up.  I'll be home early today, because I'm going to
help Bruno's live, then I'll upload him to github

Em seg, 27 de jul de 2020 10:43, Charles Oliveira <notifications@github.com>
escreveu:

**Comment by chaws** [comment]
ack
